src/Panel/Panel.php

```diff
@@ -276,18 +279,50 @@ public static function data(array $data = []): array
             }
         ];
 
+        // check for explicitly requested globals to be included
```

Comment:
As mentioned above, the duality of `include` and `only` isn't really clear to me - they seem to do different, but I don't get exactly what. And their names are too equal to carry meaning for differentiation.

Code here looks like a lot of duplicate code from the `only` implementation, would be better to have this DRY.